Remove the Singleton functionality of the OOReact Bridge

lental opened the pull request:
Changed ReactBridge header to be all instance methods
Use instance properties instead of static RCTBridge and SkinViewController
add ViewController Event method for non-Controller classes to send events (instead of passing around an OOReactBridge)
OOUpNextManager doesn't need any ViewController functionality, so in that case I just pass the bridge. (I can be convinced of something better here)
VolumeManager really has nothing to do with message sending, move that into the ViewController

lental committed:
Remove the Singleton functionality of the OOReact Bridge

brol1dev commented on sdk/iOS/OoyalaSkinSDK/OOReactBridge.m:
```diff
@@ -22,12 +22,15 @@
 #import "OOConstant.h"
 
+@interface OOReactBridge()
+@property OOSkinViewController *controller;
```
do we want to add nonatomic attribute?

brol1dev reviewed (approved):
Super nice :) Code looks better!

lental committed:
Fixed a memory leak with SkinviewController

The leak caused SkinViewController, and therefore everything else, to be retained

brol1dev commented on sdk/iOS/OoyalaSkinSDK/OOSkinViewController.m:
```diff
@@ -74,9 +74,13 @@ - (instancetype)initWithPlayer:(OOOoyalaPlayer *)player
     self.ooBridge = [OOReactBridge new];
+    //Passing self.ooBridge itself in the anonymous function counts as a self strong reference.
+    //I create a copy of the pointer to avoid that
+    OOReactBridge *newBridge = self.ooBridge;
```
Oh! Thanks for sharing this. Good to know :)
